sql/src/main/java/org/elasticsearch/action/bulk/BulkShardProcessor.java

```diff
@@ -121,11 +123,16 @@ public BulkShardProcessor(ClusterService clusterService,
         this.bulkSize = bulkSize;
         this.createIndicesBulkSize = Math.min(bulkSize, MAX_CREATE_INDICES_BULK_SIZE);
 
-        this.autoCreateIndex = new AutoCreateIndex(settings);
```

Comment:
no, it is now disabled, but for partitions we want to use the AutoCreateIndex class to determine if we need to create that index. so we must enable it for that case.
but this new Settings class can be static, will change that